linux.d/arch

```diff
@@ -27,6 +27,9 @@ export REQUIRED_DEV_PACKAGES=(
     wget
 )
 
+# Arch has switched to cmake 4.x, pretend it didn't
```

Comment:
What if we just move this to the buildlinux.sh instead? There shouldn't be side effect defining this when using cmake 3.x right? It'll just be ignored.

Reply:
As I mentioned in the PR message I'm not sure and I have no way of checking all edge cases. If you are comfortable with moving it I'll do it.

Reply:
`CMAKE_POLICY_VERSION_MINIMUM` is introduced in cmake 4.0, so older versions won't do anything about it. I think it should be safe to do so.

Reply:
OK, moved.